simplifying the Zone group explanation for Local Zones OSDOCS-5240 (#3)
maxwelldb/openshift-docs: modules/installation-aws-add-local-zone-locations.adoc

<1> For `<region_name>`, specify a valid AWS region name, such as `us-east-1`.

. Review the list of zones that your region contains by running the following command:

> List the zones available in your region by running the following command:

Reviewer: ```suggestion
List the zones that are available in your region by running the following command:
```

+
[source,terminal]
----
$ aws --region ${CLUSTER_REGION} ec2 describe-availability-zones \
    --query 'AvailabilityZones[].[{ZoneName: ZoneName, GroupName: GroupName, Status: OptInStatus}]' \
    --filters Name=zone-type,Values=local-zone \
    --all-availability-zones
----
+
Depending on the region, the list of available zones can be long. The command will return the following fields:
+

> `ZoneName`:: The name of the Zone.

Reviewer: ```suggestion
`ZoneName`:: The name of the Local Zone.
```

> `GroupName`:: The group that the zone is part of. You need to save this name to opt-in.

Reviewer: ```suggestion
`GroupName`:: The group that the zone is part of. You need to save this name to opt in.
```

> `Status`:: The status of the Zone Group. If the status is `not-opted-in` you must opt-in the `GroupName` running the commands below.

Reviewer: ```suggestion
`Status`:: The status of the Local Zone group. If the status is `not-opted-in`, you must opt in the `GroupName` by running the commands that follow.
```

. Export a variable to contain the name of the Local Zone to host your VPC by running the following command:
+
[source,terminal]
----

> $ export ZONE_GROUP_NAME="<value of GroupName>" <1>

Reviewer: ```suggestion
$ export ZONE_GROUP_NAME="<value_of_GroupName>" <1>
```

----

> <1> The `<value of GroupName>`, must be replaced by the name of the group of the Local Zone you want to create subnets. For example, specify `us-east-1-nyc-1` to use the zone `us-east-1-nyc-1a`, US East (New York).

Reviewer: ```suggestion
+
where:

<value_of_GroupName>:: Specifies the name of the group of the Local Zone you want to create subnets on. For example, specify `us-east-1-nyc-1` to use the zone `us-east-1-nyc-1a`, US East (New York).
```
